### ISSUE TITLE
MoveToMapPokemon: error on came back to last position

### ISSUE THREAD

**Issue by Lunik** [PokemonGoF/PokemonGo-Bot#3016]
I get this error when the bot try to get back to his last position after snipe.

### Actual Behavior
```
2016-08-08 09:59:25,902 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_teleport_to] Tel
eporting to Tentacruel. (0.36km)
2016-08-08 09:59:25,902 [MoveToMapPokemon] [INFO] [move_to_map_pokemon_encounter] Encou
ntered Pokemon: Tentacruel
Traceback (most recent call last):
  File "pokecli.py", line 503, in <module>
    main()
  File "pokecli.py", line 89, in main
    bot.tick()
  File "/home/lunik/pkm/pokemongo_bot/__init__.py", line 434, in tick
    if worker.work() == WorkerResult.RUNNING:
  File "/home/lunik/pkm/pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 255, i
n work
    return self.snipe(pokemon)
  File "/home/lunik/pkm/pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 214, i
n snipe
    self._teleport_back()
  File "/home/lunik/pkm/pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 339, i
n _teleport_back
    data={'last_lat': last_position[0], 'last_lon': last_position[1]}
  File "/home/lunik/pkm/pokemongo_bot/base_task.py", line 27, in emit_event
    data=data
  File "/home/lunik/pkm/pokemongo_bot/event_manager.py", line 61, in emit
    formatted_msg = formatted.format(**data)
KeyError: 'last_long'
```

### Steps to Reproduce
```
"type": "MoveToMapPokemon",
        "config": {
          "address": "http://localhost:5050",
          "max_distance": 500,
          "min_time": 60,
          "prioritize_vips": true,
          "snipe": true,
          "update_map": false,
          "mode": "priority",
          "catch": {}
        }
```

### Other Information
OS: Ubuntu
Git Commit: 563f898f61233ff86d38d83b68d8399c1f63051b
Python Version: Python 2.7.9

**Comment by mikaelkristiansson** [comment]
I had the same problem this morning, so I just reversed to this commit: https://github.com/PokemonGoF/PokemonGo-Bot/tree/e5b7eda6cd7054baadbd6440bf9b94d3d5cd88e3 until it will be fixed

**Comment by Raiven66** [comment]
sorry for the offtopic question on this thread but what PoGO Map are you using now?

**Comment by Lunik** [comment]
https://github.com/PokemonGoMap/PokemonGo-Map

**Comment by mhdasding** [comment]
will be fixed with #3014
